controllers/cron_controller.py

```diff
@@ -666,6 +689,29 @@ def get(self):
             logging.exception(e)
 
 
+class CheckMediaDo(webapp.RequestHandler):
+    """
+    Remove unavailable media
+    """
+    def get(self, media_key):
+        media = ndb.Key(Media, media_key).get()
+
+        if media.is_image:
+            url = media.view_image_url
+        elif media.is_social:
+            url = media.social_profile_url
+        elif media.media_type_enum == MediaType.YOUTUBE_VIDEO:
+            url = "https://www.youtube.com/oembed?url={}&format=json".format(media.youtube_url_link)
+        elif media.media_type_enum == MediaType.EXTERNAL_LINK:
+            url = media.external_link
+
+        if not WebsiteHelper.exists(url):
+            media.key.delete()
+        else:
+            media.checked = datetime.datetime.now()
+            media.put()
```

Comment:
Add the following comment here:

`# .put() is being used instead of MediaManipulator to avoid cache busting.`